__tests__/app.test.js

```diff
@@ -140,17 +140,17 @@ describe('POST: api/articles/:article_id/comments', () => {
             expect(body.msg).toBe('Bad request');
           });
       });
-      test('POST:400 responds with an appropriate error message when provided with an invalid post object value data type', () => {
+      test('POST:404 responds with an appropriate error message when provided with an invalid post object value data type', () => {
```

Comment:
I think there is a mild confusion because there are so many errors, but an invalid data type to search by, in this case a number 1 on a string column, is absolutely a 400.

Reply:
the problem I have is psql is throwing the same error code (23503) for both incorrect data type test and valid but non existing article test. I dont know how handle them separately on a 400 and 404 for each.